### PR TITLE
Update README with canvas user instructions

### PR DESCRIPTION
**Pull request by jessouyangbraven**
**Task**: https://app.asana.com/0/1174274412967132/1187084445259049

**Description**: We need users in Canvas and our development environment to have matching emails and IDs, add instructions to README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,11 +105,26 @@ first ask someone to create a Salesforce account for you (or get the admin accou
...
+### Canvas
 
 #### Create A Portal Course
-Create a course on the Portal as your Sandbox/Dev Env. Call it `Playground - <insert your name>`
+Create a course in Canvas as your sandbox/development environment. Call it `Playground - <insert your name>`. Publish it. 
+
+#### Create test users
+In your Canvas course, select **People** in navigation and add test users to your course by email. 
+
+The format for the email is to your Braven email address, with `+teststudent1` , `+testta1`, etc. appended.
```
Comment (sadleb) [inline README.md:116]: Can you put an example? E.g. `myname+teststudent1@bebraven.org` ?

File: README.md [hunk 2]
```diff
@@ -105,11 +105,26 @@ first ask someone to create a Salesforce account for you (or get the admin accou
...
+
+Once you've added the test users, check your email, and accept the course invitations.
```
Comment (sadleb) [inline README.md:118]: You can also click on the user in the People tab and choose "Act as User" and the invites will either be accepted automatically or you may have to click Accept.
image

File: README.md [hunk 3]
```diff
@@ -105,11 +105,26 @@ first ask someone to create a Salesforce account for you (or get the admin accou
...
+Create a course in Canvas as your sandbox/development environment. Call it `Playground - <insert your name>`. Publish it. 
+
+#### Create test users
+In your Canvas course, select **People** in navigation and add test users to your course by email. 
+
+The format for the email is to your Braven email address, with `+teststudent1` , `+testta1`, etc. appended.
+
+Once you've added the test users, check your email, and accept the course invitations. 
+
+In your development directory, run `devc` and add your test user, editing the **email** and **canvas_id** to match:
```
Comment (sadleb) [inline README.md:120]: Can you add instructions for how to easily get the **canvas_id**? E.g. hover over the person's name in the Person tab and the ID will be in the URL. E.g. `78` in this example `https://braven.instructure.com/courses/48/users/78`

File: README.md (hunk 2 above)
Comment (jessouyangbraven) [reply README.md]: I feel like the README would benefit from screenshots. For future reference, is there a way to link images?

Comment (rshipp) [reply README.md]: If they're uploaded somewhere or in the repo.

**Pull request merged**